Main.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+import (
+  "geoffrey/commands"

```

Comment:
I'm not sure how this is working. I don't see a package named `geoffrey`. On my IDE it is happy when I do `./commands` but thats still not proper form.

go project structures love the long form that you see in repository urls. So `github.com/mmoghaddam385/GeoffreyBot/commands` would be the proper way. 

Of course for this to work properly on your machine, or any machine that is running this, the `GOPATH` would need to be set and this code would need to live in `$GOPATH/github.com/mmoghaddam385/`